etl/steps/data/meadow/un/2024-06-24/sdg_regions.py

```diff
@@ -0,0 +1,33 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("sdg_regions.csv")
+
+    # Load data from snapshot.
+    tb = snap.read(delimiter=";")
+
+    #
+    # Process data.
+    #
+    # Ensure all columns are snake-case, set an appropriate index, and sort conveniently.
+    # tb = tb.format(["Country or Area"])
```

Comment:
just curious - do you know why tb.format doesn't work here?

Reply:
I'm not sure tbh, it happens occasionally for me, I've [reported it here](https://github.com/owid/etl/issues/2874)